### PR TITLE
chore: cleanup Prisma Client extensions Preview feature flag mentions

### LINKED ISSUE #4880: Remove Prisma Client extensions Preview feature flag

### PR DESCRIPTION
**Pull request by ruheni**
Closes #4880

### CODE REVIEW

**Comment by janpio** [comment]
Unfortunately this should now mention that they _were_ in preview since and until in at least some places so users of older Prisma versions can still discover that information to be able to use it in their projects.

**Comment by ruheni** [comment]
That will be mentioned. The PR is still a draft 😄

**Comment by ruheni** [comment]
I've updated the pages with admonitions of when it got into Preview & General Availability (cc @janpio). Let me know if it's what you had in mind.

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx [hunk 1]
```diff
@@ -7,7 +7,13 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add custom methods to your models. We introduced this feature in version 4.7.0.
+<Admonition type="info">
+
+Prisma Client extensions are [Generally Available]() from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx]: missing link

**Comment by janpio** [comment]
Yep, that looks like what is needed 👍

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx (hunk 1 above)
Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx]: I meant to remove the link there. I'll update this.

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx [hunk 2]
```diff
@@ -7,20 +7,15 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `client` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component to add top-level methods to Prisma Client. We introduced this feature in version 4.7.0.
+<Admonition type="info">
 
-</TopBlock>
+Prisma Client extensions are [Generally Available]() from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx]: empty link

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx [hunk 3]
```diff
@@ -7,22 +7,17 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `query` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to hook into the query life-cycle and modify an incoming query or its result. We introduced this feature in version 4.7.0.
+<Admonition type="info">
 
-You can use Prisma Client extensions `query` component to create independent clients. This provides an alternative to [middlewares](/concepts/components/prisma-client/middleware). You can bind one client to a specific filter or user, and another client to another filter or user. For example, you might do this to get [user isolation](/concepts/components/prisma-client/client-extensions#extended-clients) in a row-level security (RLS) extension. In addition, unlike middlewares the `query` extension component gives you end-to-end type safety. [Learn more about `query` extensions versus middlewares](#query-extensions-versus-middlewares).
+Prisma Client extensions are [Generally Available]() from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx]: empty link

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/130-result.mdx [hunk 4]
```diff
@@ -7,20 +7,15 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `result` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add custom fields and methods to query results. We introduced this feature in version 4.7.0.
+<Admonition type="info">
 
-</TopBlock>
+Prisma Client extensions are [Generally Available]() from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/130-result.mdx]: empty link

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
-preview: true
 tocDepth: 4
 ---
 
 <TopBlock>
 
-From version 4.7.0, you can use Prisma Client extensions to add functionality to your models, result objects, and queries, or to add client-level methods.
+<Admonition type="info">
+
+Prisma Client extensions are [Generally Available]() from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx]: empty link

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx (hunk 2 above)
Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx]: I've removed all the empty links

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx [hunk 6]
```diff
@@ -7,7 +7,13 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add custom methods to your models. We introduced this feature in version 4.7.0.
+<Admonition type="info">
+
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx]: missing the preview feature flag it was introduced under so people can figure out how to use it with older versions

Comment (ruheni) [reply content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx]: Done!

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx [hunk 7]
```diff
@@ -7,7 +7,13 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `model` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add custom methods to your models. We introduced this feature in version 4.7.0.
+<Admonition type="info">
+
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/100-model.mdx]:
```suggestion
Prisma Client extensions are Generally Available from versions 4.16.0 and later. They were introduced in Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx [hunk 8]
```diff
@@ -7,20 +7,15 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `client` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component to add top-level methods to Prisma Client. We introduced this feature in version 4.7.0.
+<Admonition type="info">
 
-</TopBlock>
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx]:
```suggestion
Prisma Client extensions are Generally Available from versions 4.16.0 and later. They were introduced in Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx [hunk 9]
```diff
@@ -7,22 +7,17 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `query` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to hook into the query life-cycle and modify an incoming query or its result. We introduced this feature in version 4.7.0.
+<Admonition type="info">
 
-You can use Prisma Client extensions `query` component to create independent clients. This provides an alternative to [middlewares](/concepts/components/prisma-client/middleware). You can bind one client to a specific filter or user, and another client to another filter or user. For example, you might do this to get [user isolation](/concepts/components/prisma-client/client-extensions#extended-clients) in a row-level security (RLS) extension. In addition, unlike middlewares the `query` extension component gives you end-to-end type safety. [Learn more about `query` extensions versus middlewares](#query-extensions-versus-middlewares).
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/120-query.mdx]:
```suggestion
Prisma Client extensions are Generally Available from versions 4.16.0 and later. They were introduced in Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/130-result.mdx [hunk 10]
```diff
@@ -7,20 +7,15 @@ tocDepth: 4
 
 <TopBlock>
 
-You can use the `result` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) component type to add custom fields and methods to query results. We introduced this feature in version 4.7.0.
+<Admonition type="info">
 
-</TopBlock>
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/130-result.mdx]:
```suggestion
Prisma Client extensions are Generally Available from versions 4.16.0 and later. They were introduced in Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```

File: content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx [hunk 11]
```diff
@@ -1,14 +1,19 @@
...
-preview: true
 tocDepth: 4
 ---
 
 <TopBlock>
 
-From version 4.7.0, you can use Prisma Client extensions to add functionality to your models, result objects, and queries, or to add client-level methods.
+<Admonition type="info">
+
+Prisma Client extensions are Generally Available from versions 4.16.0 and later. It was introduced into Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/260-client-extensions/index.mdx]:
```suggestion
Prisma Client extensions are Generally Available from versions 4.16.0 and later. They were introduced in Preview in version 4.7.0. Make sure you enable the `clientExtensions` Preview feature flag if you are running on a version earlier than 4.16.0.
```

**Review by janpio — APPROVED** [review]

**Review by jharrell — APPROVED** [review]

**Pull request merged**